### PR TITLE
Hyundai: add Genesis G90 2017 FW versions

### PR DESCRIPTION
**Pull request by gribneau**
Fingerprint for 2017 Genesis G90 Ultimate 5.0 AWD

Test Route ID
8ccda2b5d847617d|2022-07-14--21-08-33

### CODE REVIEW

**Comment by gribneau** [comment]
Here are test drives with CAR.GENESIS_G90 removed from the 255 steering torque blocklist:

https://connect.comma.ai/8ccda2b5d847617d/1657993211825/1657994895826

8ccda2b5d847617d|2022-07-16--13-40-11

8ccda2b5d847617d|2022-07-16--13-30-17

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ def __init__(self, CP):
 
     # To determine the limit for your car, find the maximum value that the stock LKAS will request.
     # If the max stock LKAS request is <384, add your car to this list.
-    elif CP.carFingerprint in (CAR.GENESIS_G80, CAR.GENESIS_G90, CAR.ELANTRA, CAR.HYUNDAI_GENESIS, CAR.ELANTRA_GT_I30, CAR.IONIQ,
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:31]: Let's do this separately, it likely requires a new tune

File: selfdrive/car/hyundai/values.py [hunk 2]
```diff
@@ -28,7 +28,7 @@ def __init__(self, CP):
...
+    elif CP.carFingerprint in (CAR.GENESIS_G80, CAR.ELANTRA, CAR.HYUNDAI_GENESIS, CAR.ELANTRA_GT_I30, CAR.IONIQ,
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:31]:
```suggestion
    elif CP.carFingerprint in (CAR.GENESIS_G80, CAR.GENESIS_G90, CAR.ELANTRA, CAR.HYUNDAI_GENESIS, CAR.ELANTRA_GT_I30, CAR.IONIQ,
```

Comment (sshane) [reply selfdrive/car/hyundai/values.py:31]: Let's do this separately, it likely requires a new tune

**Comment by sshane** [comment]
Thanks for the PR! Feel free to open a new one to remove your car from the blacklist

**Comment by gribneau** [comment]
> Thanks for the PR! Feel free to open a new one to remove your car from the blacklist

Thanls @sshane.

I've just completed as 1200 mile trip with the 255 blocklist removed.

I have a very good success case - fairly sharp bends on i65 heading south into Bardstown KY. The car handled that nicely at 70+.

I also have a good failure case - sharper bends on i24 heading south east into Chattanooga TN. Many of those exceeded the steering capabilities and required manual intervention.

I'll connect the c3 via hotspot so it can return that data. Is there any other information from those routes that would be useful, or is the automatic data gathering enough?

**Pull request merged**